src/pages/VaultsPage/DepositModal.tsx

```diff
@@ -52,8 +51,10 @@ export default function DepositModal({
 
     (async () => {
       const tokenAddress =
-        vault.type === 'OTHER' ? vault.simpleTokenAddress : vault.tokenAddress;
-      const token = selectedContracts.getERC20(tokenAddress);
+        vault.symbol === 'OTHER'
```

Comment:
in the previous implementation, type & symbol were separate fields... if we're combining them (see my comment about useVaults), this check might have to become e.g. `Object.values(Asset).includes(vault.symbol)`

Reply:
Done as well.